static/app/views/performance/browser/resources/resourceSummaryPage/index.tsx

```diff
@@ -57,10 +57,7 @@ function ResourceSummary() {
     ],
   });
   const spanMetrics = data[0] ?? {};
-
-  const isImage = IMAGE_FILE_EXTENSIONS.includes(
-    spanMetrics[SpanMetricsField.SPAN_DESCRIPTION]?.split('.').pop() || ''
-  );
+  const isImage = spanOp === 'resource.img';
```

Comment:
l: Ideally `resource.img` is extracted to a constant

Reply:
done!